qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowserplotinteraction.py

```diff
@@ -164,7 +164,11 @@ def remove_guess_all(self):
         """
         if self.guess_all_line is None:
             return
-        self.guess_all_line.remove()
+        try:
+            self.guess_all_line.remove()
+        except ValueError:
```

Comment:
Are we certain that this is the only way of producing a value error in this case?

Reply:
I would say so. This is the only call that has the potential to throw. My understanding is that there will a list of artists in matplotlib and when we call self.guess_all_line.remove() it equates to calling artist_list.remove(self.guess_all_line), but this line has already been removed.

Reply:
That's ok then. In that case just resove the merge conflicts and this will be good.